docs/src/man/getting_started.md

```diff
@@ -23,6 +23,21 @@ Nullable()
 Nullable(1) + Nullable()
 ```
 
+The get() function can be used to extract the value from the `Nullable` wrapper.
```

Comment:
Ah, and missing backticks around `get`. Also I think in general the parentheses are omitted.